releng/raw-binaries/Dockerfile

```diff
@@ -43,11 +49,19 @@ RUN mkdir -p /opt/osxcross && \
 
 ENV PATH="/opt/osxcross/target/bin:${PATH}"
 
-# Wrapper to pull a version of pkg-config at runtime
-ENV PKG_CONFIG=/usr/local/bin/pkg-config.sh
+RUN cd /tmp && \
+    git clone https://github.com/richfelker/musl-cross-make.git && \
```

Comment:
How reliable and secure do we believe this tool is? Do we have a vetting process for raw code dependencies like this? Are there any packaged alternatives?

Reply:
I copied this from the 2.x build - @danxmoran is probably the best person to ask about what security reviews this went through (if any).

Reply:
* No security review
* No vetting process that I know of
* There is a packaged alternative at https://musl.cc/, but it's not stable:
   * They only publish "latest" builds of each toolchain, so you can't pin a version
   * When they rebuild things, they start by clearing out all current downloads (so you can hit 404s)

I basically went with "what does the community use" when searching for options. I'd love to switch to native ARM builds using Circle's ARM executors, and nuke this piece from orbit.

Reply:
Hello. You can certainly pin musl.cc toolchains, e.g. https://more.musl.cc/10-20210301/i686-linux-musl/. Only the homepage is volatile.

As far as rebuilds go, I try to avoid clearing out existing binaries; the reason many of the old ones do not exist now is because of a catastrophic disk failure. The configuration files (to reproduce toolchains from source) since mid-2019 are still available if you need a particular version.